example/install_src.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+PWD=$(pwd)
+MYDIR=${1:-$PWD}
+if [ ! -d $MYDIR ]; then
+    echo "[Error] working directory=$MYDIR doesn't exist"
+    exit -1
+fi
+
+pushd $MYDIR
+
+WORKDIR=$(pwd)  # Absolute Path
+
+
+set -e
+set -x
+
+BRANCH=${2:-master} # can be a tag like 1.0.0
+
+
+
+NODE_CORE_REPOS=("on-core" "on-tasks" "di.js" )
+NODE_OTHER_REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" )
```

Comment:
it's more clear that it's named 'RACKHD_MAIN_REPOS', or ‘RACKHD_CORE_REPOS’(above named as 'RACKHD_DEP_REPOS')